api/base/serializers.py

```diff
@@ -301,6 +301,20 @@ def should_hide(self, instance):
         return True
 
 
+class ShowIfChronosSubmitter(ConditionalField):
+    """
+    If the ChronosSubmission instance's submitter is not the current user, hide this field.
+    """
+
+    def should_show(self, instance):
+        request = self.context.get('request')
+        auth = utils.get_user_auth(request)
+        if auth.logged_in:
+            if instance.submitter == auth.user:
+                return True
+        return False
+
+
```

Comment:
Since this is only relevant for the Chronos serializers, should it be defined in `api/chronos/serializers.py` instead of in base? Or are we wanting to keep all of these `ConditionalField` classes in base? @pattisdr?

Reply:
Hmm, I feel like this one could go either way since we seem to have lots of ConditionalFields in `base` with one use-case. But this one is super-specific to Chronos though, so I'd lean to putting it in `api/chronos/serializers.py` instead, and in the future, if it is definitely applicable to just one serializer, locating it more specifically with that serializer.

Reply:
Hmm, I feel like this one could go either way since we seem to have lots of ConditionalFields in `base` with one use-case. But this one is super-specific to Chronos though, so I'd lean to putting it in `api/chronos/serializers.py` instead, and in the future, if it is definitely applicable to just one serializer, locating it more specifically with that serializer.

Reply:
Hmm, I feel like this one could go either way since we seem to have lots of ConditionalFields in `base` with one use-case. But this one is super-specific to Chronos though, so I'd lean to putting it in `api/chronos/serializers.py` instead, and in the future, if it is definitely applicable to just one serializer, locating it more specifically with that serializer.